### PR TITLE
[prometheus]enable hostAliases to be defined for alermanager pod

### PR DESCRIPTION
**Pull request by slavonicsniper**
#### What this PR does / why we need it:
so far hostaliases can be defined only in the server section for prometheus pods, this change adds the same for alertmanager pods
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Comment by zanhsieh** [comment]
@slavonicsniper 
Could you bump the chart version to latest please?

**Review by monotek — DISMISSED** [review]

**Comment by zanhsieh** [comment]
@slavonicsniper 
Could you bump the chart version to latest please?

File: charts/prometheus/templates/alertmanager/deploy.yaml [hunk 1]
```diff
@@ -112,6 +112,10 @@ spec:
     {{- if .Values.alertmanager.nodeSelector }}
       nodeSelector:
 {{ toYaml .Values.alertmanager.nodeSelector | indent 8 }}
+    {{- end }}
+    {{- if .Values.alertmanager.hostAliases }}
+      hostAliases:
+{{ toYaml .Values.alertmanager.hostAliases | indent 8 }}
```
Comment (naseemkullah) [inline charts/prometheus/templates/alertmanager/deploy.yaml:118]:
```suggestion
        {{- toYaml .Values.alertmanager.hostAliases | nindent 8 }}
```

nit

**Comment by monotek** [comment]
@slavonicsniper 
I've added the chart version in the github frontend and alos added @naseemkullah requested changes.
Unfortunately the dco check fails now.
As i guess i 'cant force push in your branch, could you please rebase und push again?

**Review by monotek — CHANGES_REQUESTED** [review]
@slavonicsniper
I've added the chart version in the github frontend and alos added @naseemkullah requested changes.
Unfortunately the dco check fails now.
As i guess i 'cant force push in your branch, could you please rebase und push again?

**Review by naseemkullah — DISMISSED** [review]

**Review by zanhsieh — APPROVED** [review]

**Comment by zanhsieh** [comment]
@slavonicsniper 
There are some DCOs you need to sign. See below "Details" next to DCO.

**Pull request closed without merging**